server/seenomaly/lambda/app.py

```diff
@@ -0,0 +1,6 @@
+import multiEventHandler 
+def handler(event, context):
+    # check event header 
+    if "httpMethod" in event and event["httpMethod"] =="POST":
```

Comment:
I think we need to have a python automatic lint (just like prettier), maybe I will work on it later.